libs/liblz4/Makefile

```diff
@@ -7,54 +7,80 @@
 
 include $(TOPDIR)/rules.mk
 
-# Although liblz4 exports a major.minor.patch version, it isn't always
-# incremented for new releases. Check the NEWS file and instead use a
-# release tag when appropriate. (eg: PKG_VERSION:=r131)
 PKG_NAME:=liblz4
-PKG_VERSION:=v1.7.5
+PKG_VERSION:=1.9.1
 PKG_RELEASE:=1
 
-PKG_LICENSE:=BSD-2-Clause
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/lz4/lz4/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=f8377c89dad5c9f266edc0be9b73595296ecafd5bfa1000de148096c50052dc4
+PKG_BUILD_DIR:=$(BUILD_DIR)/lz4-$(PKG_VERSION)
+
 PKG_MAINTAINER:=Darik Horn <dajhorn@vanadac.com>
+PKG_LICENSE:=BSD-2-Clause GPL-2.0-only
```

Comment:
I don't know how pedantic we are going to be with regards of license information.   The library is licensed under the BSD-2-Clause, and the app is GPL-2.0 or later (look at the preamble in files, such as `programs/lz4cli.c`, including the "any later version" text.

IMO, we can use `LICENSE:=BSD-2-Clause`; `LICENSE_FILES:=LICENSE lib/LICENSE` under `Package/liblz4`, and `LICENSE=GPL-2.0-or-later`; `LICENSE_FILES:=LICENSE programs/COPYING` under `Package/lz4`.

Reply:
I like this. edited.